collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -101,7 +101,6 @@
 import org.odk.collect.android.formentry.repeats.DeleteRepeatDialogFragment;
 import org.odk.collect.android.formentry.saving.FormSaveViewModel;
 import org.odk.collect.android.formentry.saving.SaveFormProgressDialogFragment;
-import org.odk.collect.android.forms.FormsRepository;
```

Comment:
Looks like your IntelliJ settings aren't set to alphabetize imports. Would be great to revert changes to files where this is the only change so they don't get touched by this PR.

Reply:
Looks like that's default it's just that Studio doesn't do that when you move code between packages. Optimizing imports fixes it. Annoying!